packages/atlas/src/views/viewer/NftPurchaseView/NftPurchaseView.tsx

```diff
@@ -0,0 +1,377 @@
+                      />
+                    </FlexWrapper>
+                  </FlexWrapper>
+                )}
+              </Header>
+              {type === 'auction' ? (
+                <>
+                  <CurrentBidWrapper>
+                    {BID ? (
+                      <ActiveBidWrapper>
+                        <ActionBarCell>
+                          <Text variant="h300" secondary spacing={{ bottom: 2 }}>
+                            Current bid
+                          </Text>
+                          <FlexWrapper>
+                            <CurrentBidAvatar size="bid" />
+                            <BidderName variant="h400">{BID.bidder.name}</BidderName>
+                          </FlexWrapper>
+                        </ActionBarCell>
+                        <ActionBarCell>
+                          <Text variant="h300" secondary spacing={{ bottom: 2 }}>
+                            Bid amount
+                          </Text>
+                          <FlexWrapper>
+                            <CurrentBidJoyToken size={24} variant="silver" />
+                            <Text variant="h400">{BID.amount}</Text>
+                          </FlexWrapper>
+                        </ActionBarCell>
+                      </ActiveBidWrapper>
+                    ) : (
+                      <ActiveBidWrapper>
+                        <ActionBarCell>
+                          <Text variant="h300" secondary spacing={{ bottom: 2 }}>
+                            Current bid
+                          </Text>
+                          <Text variant="h400">Nobody has bid yet</Text>
+                        </ActionBarCell>
+                      </ActiveBidWrapper>
+                    )}
+                  </CurrentBidWrapper>
+                  <MinimumBidWrapper>
+                    <MinimumBid>
+                      <Text variant="h300" secondary>
+                        Minimum bid:
+                      </Text>
+                      <JoyTokenIcon variant="silver" size={24} /> <Text variant="h400">{MINIMUM_BID}</Text>
+                    </MinimumBid>
+                    <div>
+                      <Text variant="t100" secondary>
+                        Buy now: {FIXED_PRICE} tJOY
+                      </Text>
+                    </div>
+                  </MinimumBidWrapper>
+                  <TextField
+                    {...register('bid', {
+                      required: true,
+                      validate: {
+                        bidTooLow: (value) =>
+                          Number(value) >= MINIMUM_BID ? true : 'Your bid must be higher than minimum bid',
+                        bidTooHigh: (value) =>
+                          accountBalance
+                            ? Number(value) + TRANSACTION_FEE > accountBalance
+                              ? 'Insufficient funds.'
+                              : true
+                            : true,
```

Comment:
Not sure if this is included in the design but we also should handle this case:
1. The auction type is English
2. The bid is lower than the current top one

In open auction we don't care since users can post bids lower than the current winning one, but in English auction you always have to outbid the top bid

Reply:
After short a discussion on rocket chat, we decided that when integrating we will set current bid as minimum bid.